test_collections/matter/sdk_tests/support/python_testing/models/utils.py

```diff
@@ -55,16 +55,28 @@ async def generate_command_arguments(
     if dut_config.trace_log:
         arguments.append("--trace-to json:log")
 
-    if not omit_commissioning_method:
+    if dut_config.enhanced_setup_flow:
+        arguments.append("--require-tc-acknowledgements 1")
+        arguments.append(
+            f"--tc-acknowledgements {dut_config.enhanced_setup_flow.tc_user_response}"
+        )
+        arguments.append(
+            f"--tc-acknowledgements-version {dut_config.enhanced_setup_flow.tc_version}"
+        )
+
+    if omit_commissioning_method:
+        arguments.append(f"--in-test-commissioning-method {pairing_mode}")
+
+    else:
         arguments.append(f"--commissioning-method {pairing_mode}")
 
-        if pairing_mode == DutPairingModeEnum.BLE_WIFI:
-            arguments.append(f"--wifi-ssid {config.network.wifi.ssid}")
-            arguments.append(f"--wifi-passphrase {config.network.wifi.password}")
+    if pairing_mode == DutPairingModeEnum.BLE_WIFI:
+        arguments.append(f"--wifi-ssid {config.network.wifi.ssid}")
+        arguments.append(f"--wifi-passphrase {config.network.wifi.password}")
```

Comment:
```suggestion
        if pairing_mode == DutPairingModeEnum.BLE_WIFI:
            arguments.append(f"--wifi-ssid {config.network.wifi.ssid}")
            arguments.append(f"--wifi-passphrase {config.network.wifi.password}")
```
Could you please check this indentation, it seems it changes the code flow the way it was before.

Reply:
@rquidute , I was getting additional CI failures due to black, so I locally ran the reformat... I think the repo is in a bad state. What should we do?